package-lock.json

```diff
@@ -3518,7 +3518,6 @@
           "resolved": "https://registry.npmjs.org/is-glob/-/is-glob-4.0.1.tgz",
           "integrity": "sha512-5G0tKtBTFImOqDnLB2hG6Bp2qcKEFduo4tZu9MT/H6NQv/ghhy30o55ufafxJ/LdH79LLs2Kfrn85TLKyA7BUg==",
           "dev": true,
-          "optional": true,
```

Comment:
Should we have this change here?

Reply:
No, we shouldn't. Second time I have this issue: haven't changed this file, haven't added it to staging, but it is in PR again. Will fix now